cyvcf2/cyvcf2.pyx

```diff
@@ -950,7 +954,7 @@ cdef class Variant(object):
             cdef int unknown = 3 if self.vcf.gts012 else 2
             for i in range(0, n * self.vcf.n_samples, n):
                 if n == 2:
-                    if gt_types[j] == unknown:
+                    if (gt_types[j] == unknown) and (self.vcf.missingness == False):
```

Comment:
sure thing!  Just pushed [the change][0].

[0]: https://github.com/brentp/cyvcf2/pull/55/commits/79173edeec45c3811b573be3b26f92ffa6a6ee97